rewrite-java/src/main/java/org/openrewrite/java/utils/ExpressionUtils.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.utils;
+
+import org.openrewrite.Tree;
+import org.openrewrite.java.tree.Expression;
+import org.openrewrite.java.tree.J;
+import org.openrewrite.java.tree.JLeftPadded;
+import org.openrewrite.java.tree.Space;
+import org.openrewrite.marker.Markers;
+import org.openrewrite.java.tree.J.Binary.Type;
+import java.util.List;
+
+public final class ExpressionUtils {
+    private ExpressionUtils() {}
+
+    /**
+     * Concat two literals to an expression with '+' and surrounded with single space.
+     */
+    public static J.Binary concatAdditionBinary(Expression left, Expression right) {
+        JLeftPadded<Type> leftPadded = new JLeftPadded<>(Space.SINGLE_SPACE,
+                Type.Addition,
+                Markers.EMPTY);
```

Comment:
I think adding a util class for this is necessary since I can see three places need this shared feature.
1. this recipe
2. recipe `ReplaceTextBlockWithString`
3. the upcoming recipe of change `StringBuild.append()` calls to `String`.

And also I change to use `JavaTemplate` to generate a `J.Binary `instead of hand-constructed like this.